.gitlab-ci.yml

```
@@ -1,8 +1,87 @@
-stages:
-  - test
+variables:
+  GOLANG_VERSION: "1.22.4"
```

(Nitpick) Using the verbose schema exposes the variable in Gitlab UI
```suggestion
  GOLANG_VERSION:
    value: "1.22.4"
    description: 'Go version to use for building and testing'
```